packages/squiggle-lang/src/rescript/Distributions/SymbolicDist/SymbolicDist.res

```diff
@@ -11,22 +11,22 @@ module Normal = {
 
   let from90PercentCI = (low, high) => {
     let mean = E.A.Floats.mean([low, high])
-    let stdev = (high -. low) /. (2. *. 1.644854)
+    let stdev = (high -. low) /. (2. *. 1.6448536269514722)
```

Comment:
where does this number come from, anyway?

Reply:
don't be afraid to put more citations in comments, I like that as a pattern (in pytorch each docstring has a bibliography).

In general I don't like to see arbitrary floats without an explanation. is it `2 * sqrt(pi)`? Idk, but I'm curious.

Reply:
Added in new commit

Reply:
Thank you.